lib/presentation/redux/states/ui_state.dart

```diff
@@ -73,26 +73,27 @@ class SearchState {
   SearchState(this.searchStatus, this.searchDestination);
 
   factory SearchState.initial() {
-    return SearchState(SearchStatus.inactive, SearchDestination.mySpace);
+    return SearchState(SearchStatus.INACTIVE, SearchDestination.MY_SPACE);
   }
 
   SearchState newSearchQuery(SearchQuery searchQuery) {
     return SearchState(searchStatus, searchDestination);
   }
 
   SearchState disableSearchState() {
-    return SearchState.initial();
+    return SearchState(SearchStatus.INACTIVE, searchDestination);
   }
 
   SearchState enableSearchState(SearchDestination searchDestination) {
-    return SearchState(SearchStatus.active, searchDestination);
+    return SearchState(SearchStatus.ACTIVE, searchDestination);
   }
 }
 
 enum SearchStatus {
-  active, inactive
+  ACTIVE, INACTIVE
 }
 
 enum SearchDestination {
-  mySpace
+  MY_SPACE,
+  SHARED_SPACE
```

Comment:
I think this should be smthg like `SHARED_SPACE_NODE`, because we also have search feature from root folder, which actually is shared space u know.